lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -164,6 +164,7 @@ export class ContentMetadataComponent implements OnChanges, OnInit {
     categoriesManagementMode = CategoriesManagementMode.ASSIGN;
     classifiableChanged = this.classifiableChangedSubject.asObservable();
     editing = false;
+    invalidProperties: string[] = [];
```

Comment:
Looking at updateInvalidProperties function you don't want to add duplicates to array if element already exists in array. Maybe better choice will be to use Set type instead of Array? It will simplify your code in couple ways, for example: 
- you don't need to manually check if array already has element - set checks it automatically and doesn't allow to add duplicates 
- you can remove element from Set based on element (no need to look for index first)

Reply:
Good idea, implemented. Thank you